CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## IN PROGRESS
 
 * Add Jest/RTL testing for `ComponentToPrint` component. Refs UICHKIN-281.
+* Switching service points is not recognized by the check-in app. Refs UICHKIN-344.
```

Comment:
When fixing a bug, the changelog and the PR title should describe the solution, i.e. what changed, not simply restate the problem. Here, something like "Retrieve service point from `okapi.currentUser` because `stripes.user` is out of sync" would be good.

Likewise, the PR description needs to explain how and why this change addresses the problem. Otherwise, the person reviewing the story has to re-troubleshoot the problem in order to assess if/how the changes address the bug.

Reply:
Fixed